website/client/src/components/appFooter.vue

```diff
@@ -790,6 +790,7 @@ import heart from '@/assets/svg/heart.svg';
 import { mapState } from '@/libs/store';
 import buyGemsModal from './payments/buyGemsModal.vue';
 import reportBug from '@/mixins/reportBug.js';
+import makeAdmin from '@/mixins/makeAdmin.js';
```

Comment:
This doesn't actually work, as makeAdmin.js is in the /libs directory instead of /mixins.